app/Forms/MembersSignupForm.php

```diff
@@ -88,7 +88,7 @@ protected function addSg() {
         $this
             ->add('nationality', 'text')
             ->add('is_singapore_pr', 'checkbox', ['label' => 'Singapore PR?'])
-            ->add('nric', 'text', ['label' => 'NRIC'])
+            ->add('nric', 'text', ['label' => 'NRIC (Upper Case)'])
```

Comment:
As a general guideline for future development (assuming the effort is not prohibitively expensive), adding a bit of automation > more documentation - it would be nice if we could add in a bit of Javascript to autocapitalise the NRIC (: but this is really a nit and the way that you approached it definitely works well - priority is definitely a working version, so don't worry about this for now.